### PR TITLE
61 fix docker build for docs

### PR DESCRIPTION
**Pull request by ywkim312**
Added env variable for PATH to docker file for pyincore build docker

### CODE REVIEW

**Comment by ywkim312** [comment]
Alternative way could be like from `RUN sphinx-build -v -b html docs/source docs/build` to `RUN /full/path/sphinx-build -v -b html docs/source docs/build`

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 - EPFDamage to remove deprecated LS/DS code and properly handle liquefaction [#32](https://github.com/IN-CORE/pyincore/issues/32)
-
+- Automatic build for pyincore documentation docker [#61](https://github.com/IN-CORE/pyincore/issues/61)
```
Comment (robkooper) [inline CHANGELOG.md:16]: add new line

**Review by robkooper — CHANGES_REQUESTED** [review]
small change

File: Dockerfile [hunk 2]
```diff
@@ -8,6 +8,7 @@ USER root
 # install packages
 WORKDIR /src
 COPY requirements.txt .
+ENV PATH="/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```
Comment (robkooper) [inline Dockerfile:11]:
```suggestion
ENV PATH "$MAMBA_ROOT_PREFIX/bin:$PATH"
```

**Review by robkooper — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (ywkim312) [reply Dockerfile]: changed

File: CHANGELOG.md (hunk 1 above)
Comment (ywkim312) [reply CHANGELOG.md:16]: added

**Review by robkooper — APPROVED** [review]

**Pull request merged**